webpack/assets/javascripts/react_app/redux/API/APIHelpers.js

```diff
@@ -0,0 +1,60 @@
+export const formatSuccess = ({ data, payload, status, error }) => ({
+  status,
+  error,
+  ...payload,
+  ...data,
+});
+
+export const formatFailure = ({ payload, status, error }) => ({
+  status,
+  error,
+  ...payload,
+});
+
+export const formatPending = ({ payload, status, error }) => ({
+  status,
+  error,
+  ...payload,
+});
+
+export const formatSuccessByPayloadKey = (response, payloadKey) => ({
+  [response.payload[payloadKey]]: formatSuccess(response),
+});
+
+export const formatFailureByPayloadKey = (failure, payloadKey) => ({
+  [failure.payload[payloadKey]]: formatFailure(failure),
+});
+
+export const formatPendingByPayloadkey = (pending, payloadKey) => ({
```

Comment:
```suggestion
export const formatPendingByPayloadKey = (pending, payloadKey) => ({
```
That lowercase k was annoying :confounded: